src/components/AssetAccountDetails.tsx

```diff
@@ -33,16 +37,40 @@ const contentPaddingY = { base: 0, md: 8 }
 export const AssetAccountDetails = ({ assetId, accountId }: AssetDetailsProps) => {
   const marketData = useAppSelector(state => selectMarketDataByAssetIdUserCurrency(state, assetId))
   const assetIds = useMemo(() => [assetId], [assetId])
+  const fetchAndUpsertCustomTokenMarketData = useFetchAndUpsertCustomTokenMarketData()
+  fetchAndUpsertCustomTokenMarketData(assetId)
 
   const assetHeader = useMemo(
     () => <AssetHeader assetId={assetId} accountId={accountId} />,
     [assetId, accountId],
   )
 
+  const isCustomAsset = useAppSelector(state => selectIsCustomAsset(state, assetId))
+
+  const MaybeCustomAssetWarning: JSX.Element | null = useMemo(
```

Comment:
suggestion: reserve title case for components

```suggestion
  const maybeCustomAssetWarning: JSX.Element | null = useMemo(
```

Reply:
Actioned in https://github.com/shapeshift/web/pull/7370/commits/45d970641b800d671d18b0b40d9f616e9afe0f6d